renderer-macos/lib/stubs/NSTextView_stubs.c

```diff
@@ -23,25 +32,79 @@ TextView *ml_NSTextView_make() {
   return txt;
 }
 
+void ml_NSTextView_applyAttributes(TextView *txt) {
+  NSRange range = NSMakeRange(0, txt.attributedString.length);
+
+  [txt.attributedString setAttributes:txt.attributedProps range:range];
+  [[txt textStorage] setAttributedString:txt.attributedString];
+}
+
+double ml_NSTextView_getTextWidth(TextView *txt) {
+  return (double)[txt.attributedString size].width;
```

Comment:
This is ok in this simple case but in general measuring strings is more complex. Any string can be rendered differently depending on how much horizontal space it's given. (something for the future though)